src/style.css

```diff
@@ -0,0 +1,137 @@
+html {
+  font-family: Roboto, Arial, sans-serif;
+  font-style: normal;
+}
+
+:root {
+  --color-hover: #00acdc;
+}
+
+body {
+  background: #e5e5e5;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  margin: 0;
+}
+
+p {
+  margin: 0;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  padding: 32px 16px 16px;
+  max-width: 198px;
+  background-color: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__container {
+  width: 166px;
+}
+
+.card__img {
+  display: block;
+  width: 160px;
+  margin: 0 3px 40px;
+  text-align: center;
+  height: 134px;
+}
+
+.card__title {
+  margin: 0 0 4px;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+
+  color: #060b35;
+}
+
+.card__product-code {
+  margin-bottom: 16px;
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: #616070;
+}
+
+/* #region product-rating*/
+.card__product-rating {
+  display: flex;
+  justify-content: space-between;
+  margin-bottom: 24px;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  margin-right: 4px;
+  width: 16px;
+  height: 16px;
+  background: url(images/star.svg) no-repeat center;
+}
+
+.stars--4 .stars__star:nth-child(-n+4) {
+  background-image: url("images/star-active.svg");
+}
+
+.card__review {
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+
+  color: #060b35;
+}
+
+/* #endregion product-rating */
```

Comment:
Remove any comments, please. 
There is no need to comment on something. Your code is self-described.